app/backend/aquifers/models.py

```diff
@@ -0,0 +1,339 @@
+UNK	Unknown
+
+    """
+    code = models.CharField(primary_key=True, max_length=3, db_column='aquifer_subtype_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'aquifer_subtype_code'
+
+    def __str__(self):
+        return self.description
+
+class AquiferProductivity(AuditModel):
+    """
+    Productivity choices for describing Aquifer 
+    -------------------
+    High
+    Low
+    Moderate
+    """
+    code = models.CharField(primary_key=True, max_length=1, db_column='aquifer_productivity_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'aquifer_productivity_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Productivity Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+class AquiferVulnerability(AuditModel):
+    """
+    Vulnerability choices for describing Aquifer 
+    -------------------
+    High
+    Low
+    Moderate
+
+    """
+    code = models.CharField(primary_key=True, max_length=1, db_column='aquifer_vulnerability_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'aquifer_vulnerability_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Vulnerability Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+class AquiferDemand(AuditModel):
+    """
+    Demand choices for describing Aquifer 
+    -------------------
+    High
+    Low
+    Moderate
+
+    """
+    code = models.CharField(primary_key=True, max_length=1, db_column='aquifer_demand_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'aquifer_demand_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Demand Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+class WaterUse(AuditModel):
+    """
+    Type of Known Water Use choices for describing Aquifer 
+    -------------------
+Domestic
```

Comment:
Insert tabs please

Reply:
Done.. although this is a placeholder comment until I can create the *.json file to populate the code table.